### ISSUE TITLE
Links to news detail in workspace do not contain a chash

### ISSUE THREAD

**Issue by bfzgaier** [georgringer/news#376]
The LinkViewHelper does not generate a chash, if used in workspace.

https://github.com/georgringer/news/blob/master/Classes/ViewHelpers/LinkViewHelper.php#L200

Trying to view the detail of a news, using such a link, will thow a 404 Error "Reason: Request parameters could not be validated (&cHash empty)" if [FE][pageNotFoundOnCHashError] is set to true.

In v8.7 true is the default for [FE][pageNotFoundOnCHashError]

**Comment by georgringer** [comment]
thanks for the issue. Can you check if links are built correctly if chash is added if `'useCacheHash'` is set to `1`?

**Comment by bfzgaier** [comment]
I have overridden the LinkViewHelper and set `'useCacheHash =1'`.
Links are build correctly and the workspace version is shown in the preview instead of the live one.

Currently I don’t see any complications with this.

We have 2 instances with about 25 editors each, who are only allowed to work in workspace mode. I did not hear any complains.

TYPO3 8.7.4,
news 6.0.0


I also did a short test for TYPO3 7.6.21 with news 6.0.0.
It seems to work with and without cHash in the links.
